cypress/fixtures/objects/errorContentObjects.js

```diff
@@ -27,3 +27,6 @@ exports.NOT_AVAILABLE = {
 exports.INVALID_TYPE_PARAMS = {
```

Comment:
If we removed 'INVALID_TYPE_PARAMS' We can probably remove this Export Also.

Reply:
review comment addresed